### PR TITLE
Use HTTP POST to mark resources and values as deleted

### LINKED ISSUE #1149: Use HTTP POST to mark a value as deleted
Marking as deleted is really an update, not a deletion, and this would allow the request to have a JSON-LD body, which would be simpler and more consistent with the rest of the API.

### LINKED ISSUE #1153: Add API v2 route for marking a resource as deleted
This could be done as a POST along with #1149.

### PR DESCRIPTION
**Pull request by benjamingeer**
Resolves #1149.
Resolves #1153.

### CODE REVIEW

File: docs/src/paradox/03-apis/api-v2/editing-resources.md [hunk 1]
```diff
@@ -243,3 +242,42 @@ If you submit a `knora-api:newModificationDate` that is earlier than the resourc
...
+    "knora-api" : "http://api.knora.org/ontology/knora-api/v2#",
+    "rdfs" : "http://www.w3.org/2000/01/rdf-schema#",
+    "xsd" : "http://www.w3.org/2001/XMLSchema#",
+    "anything" : "http://0.0.0.0:3333/ontology/0001/anything/v2#"
+  }
+}
+```
+
+The optional property `knora-api:deleteComment` specifies a comment to be attached to the
+resource, explaining why it has been marked as deleted.
```
Comment (gfoo) [inline docs/src/paradox/03-apis/api-v2/editing-resources.md]: Maybe add the response type/content ?

File: docs/src/paradox/03-apis/api-v2/editing-values.md [hunk 2]
```diff
@@ -320,11 +320,27 @@ that they do not appear in normal query results.
...
+  },
+  "@context" : {
+    "knora-api" : "http://api.knora.org/ontology/knora-api/v2#",
+    "anything" : "http://0.0.0.0:3333/ontology/0001/anything/v2#"
+  }
+}
+```
+
+The optional property `knora-api:deleteComment` specifies a comment to be attached to the
+value, explaining why it has been marked as deleted.
```
Comment (gfoo) [inline docs/src/paradox/03-apis/api-v2/editing-values.md]: 1. Maybe add the response type/content ? 
2. It breaks the API too, is `MAJOR:` means that?

Comment (subotic) [reply docs/src/paradox/03-apis/api-v2/editing-values.md]: Yes, MAJOR denotes breaking changes.

File: docs/src/paradox/03-apis/api-v2/editing-resources.md (hunk 1 above)
Comment (benjamingeer) [reply docs/src/paradox/03-apis/api-v2/editing-resources.md]: Done, thanks.

**Comment by benjamingeer** [comment]
@subotic Thanks for the review!

**Pull request merged**